k8s/showcase/vineyard-mars-pytorch/prepare-data/prepare-data.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=django-not-configured
```

Comment:
Don't do that. The option could be added to `.pylintrc` instead.

Leaving a `pylint disable` pragam at the first line of a python file looks quite wired.

Reply:
Got it.